### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Engy-Mohamed**
Thank you in advance

### CODE REVIEW

File: app/Mail/contactUs.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+use Illuminate\Mail\Mailables\Address;
+
+class contactUs extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     */
+    public function __construct( protected $data)
```
Comment (Nora-Lootfy) [inline app/Mail/contactUs.php:20]: Wow, I like that you know this syntax,,

this means you have read the release notes of PHP 8.x

File: app/Mail/contactUs.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    /**
+     * Get the message content definition.
+     */
+    public function content(): Content
+    {
+        return new Content(
+            markdown: 'mail.contactUs',
+            with:['message'=>$this->data['message'],
```
Comment (Nora-Lootfy) [inline app/Mail/contactUs.php:44]: is this message variable not causing a problem?

**Review by Nora-Lootfy — APPROVED** [review]
All looking good, but i have to check the variable message.. i remember it causes some errors when using this variable in blade files as it reserved for displaying messages

like
```php
@error('name')
{{$message}}
@enderror
```

thanks ya @Engy-Mohamed

**Comment by Engy-Mohamed** [comment]
I had thought the same as you but it worked well

**Comment by Nora-Lootfy** [comment]
Oh thanks ya @Engy-Mohamed

**Pull request merged**